lib/fusion_auth/identity_provider.ex

```diff
@@ -185,18 +184,47 @@ defmodule FusionAuth.OpenIdConnect do
 
   For more information, visit the FusionAuth API documentation for [Complete an OpenID connect login](https://fusionauth.io/docs/v1/tech/apis/identity-providers/openid-connect#complete-an-openid-connect-login)
   """
-  @spec complete_openid_connect_login(client(), login_request_body()) :: result()
-  def complete_openid_connect_login(client, login_request_body) do
+  @spec complete_identity_provider_login(client(), map()) :: result()
+  def complete_identity_provider_login(
+        client,
+        %{
+          "applicationId" => _application_id,
+          "data" => %{
+            "code" => _code,
+            "redirect_uri" => _redirect_uri
```

Comment:
Is this payload supposed to be `snake_case` here but `camelCase` below with `samlResponse`?

Reply:
so this is actually the openid login, and for some reason FA sometimes uses `snake_case` for their attributes. https://fusionauth.io/docs/v1/tech/apis/identity-providers/openid-connect#complete-an-openid-connect-login